yogstation/code/datums/components/storage/storage.dm

```diff
@@ -98,7 +98,7 @@
 		log_game("[key_name(user)] detonated a bag of holding at [loc_name(loccheck)].")
 		
 		var/obj/singularity/ohshit = new /obj/singularity(loccheck)
```

Comment:
```suggestion
		var/obj/singularity/ohshit = new /obj/singularity(loccheck, 800)
```
Singularity initialize actually supports spawning a singularity with a different amount of energy, although even a size 1 singulo shouldn't immediately disappear unless it eats a boh and/or explodes

Reply:
Boh gets deleted first so singulo won't be exploded